src/main/java/com/rcv/GuiContext.java

```diff
@@ -16,13 +16,19 @@
 
 package com.rcv;
 
+import javafx.stage.FileChooser;
+import javafx.stage.Stage;
+
 class GuiContext {
 
   // context instance
   private static final GuiContext INSTANCE = new GuiContext();
   // currently-loaded tabulator config
   private ContestConfig config;
 
+  public Stage mainWindow;
+  public FileChooser fileChooser;
```

Comment:
You're going to want to put these two inside the GuiContext; basically replicate how config works currently in here.